### PR TITLE
Webpack config

### PR DESCRIPTION
**Pull request by zenby**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,15 +2,40 @@
...
     "express": "^4.16.4",
     "socket.io": "^2.0.4"
+  },
+  "devDependencies": {
+    "babel-core": "^6.26.3",
+    "babel-loader": "^7.1.5",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-stage-3": "^6.24.1",
+    "clean-webpack-plugin": "^1.0.1",
+    "css-loader": "^2.1.0",
```
Comment (vvscode) [inline package.json:26]: not sure about heroku - does it install dev dependencies ?

File: src/css/style.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+html {
```
Comment (vvscode) [inline src/css/style.css]: looks like it worth to read about CSS-best practicies )

File: src/js/index.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
+import io from 'socket.io-client';
```
Comment (vvscode) [inline src/js/index.js]: I strongly recommend you to separate client/server side code

this file is for what?

File: src/css/style.css (hunk 2 above)
Comment (zenby) [reply src/css/style.css]: It's not the current aim of this PR 😄 
But thanks anyway.

File: src/js/index.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+const socket = io();
+const button = document.querySelector('#send-button');
+const messageInput = document.querySelector('.message_input');
+const userInput = document.querySelector('.user_input');
+const messages = document.querySelector('#messages');
+const roomInput = document.querySelector('.room_container>select');
+
+let userID;
+const storageName = 'sessionStorage';
+if (window[storageName].userID) {
```
Comment (vvscode) [inline src/js/index.js]: `window[storageName]  && window[storageName].userID` will be better, just for case of typo above

TS is the better solution )

File: src/js/index.js (hunk 3 above)
Comment (zenby) [reply src/js/index.js]: This file is for client

**Review by vvscode — APPROVED** [review]

File: src/js/index.js (hunk 4 above)
Comment (zenby) [reply src/js/index.js]: TS could be my next step, thanks.

File: src/js/index.js (hunk 3 above)
Comment (vvscode) [reply src/js/index.js]: usually we have separate directories for client / server side code

Comment (zenby) [reply src/js/index.js]: for server I have separate folder `server` here

File: package.json (hunk 1 above)
Comment (zenby) [reply package.json:26]: It seems like yes, because at current stage everything works fine.

**Pull request merged**